### PR TITLE
Add support for diff-hl

### PR DESCRIPTION
**Pull request by j0ni**
Copy the git-gutter colors for consistency.

### CODE REVIEW

**Review by monkeyjunglejuice — APPROVED** [review]

**Comment by j0ni** [comment]
I rebased this, and I see that you approved it, but I can't merge it because I don't have write access.

Anyways, it's ready to merge :)

**Pull request merged**
